common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -120,6 +126,41 @@ def _cohort_count(self, label):
         """
         return int(label.split(' (')[1].split(')')[0])
 
+    def save_cohort_settings(self):
+        """
+        Click on Save button shown after click on Settings tab or when we add a new cohort.
+        """
+        self.q(css=self._bounded_selector("div.form-actions .action-save")).first.click()
+
+    @property
+    def is_assignment_settings_disabled(self):
+        """
+        Check if assignment settings are disabled.
+        """
+        attributes = self.q(css=self._bounded_selector('.cohort-management-assignment-type-settings')).attrs('class')
+        if 'is-disabled' in attributes[0].split():
+            return True
+
+        return False
+
+    @property
+    def assignment_settings_message(self):
+        """
+        Return assignment settings disabled message in case of default cohort.
+        """
+        query = self.q(css=self._bounded_selector('.copy-error'))
+        if query.present:
+            return query.text[0]
+        else:
+            return ''
+
+    @property
+    def cohort_header_title(self):
+        """
+        return cohort header title.
```

Comment:
Nit: please clarify what the cohort header title is. For instance, does it contain the description of the cohort type? Or just  name of cohort plus number of students in cohort?